RELEASENOTES.md

```diff
@@ -94,7 +94,7 @@
   ([#3630](https://github.com/google/ExoPlayer/issues/3630)).
 * DASH:
   * Support in-band Emsg events targeting the player with scheme id
-    "urn:mpeg:dash:event:2012" and scheme values "1", "2" and "3".
+    "urn:mpeg`:dash:`event:2012" and scheme values "1", "2" and "3".
```

Comment:
Won't this result in "urn:mpeg`:dash:`event:2012"?

If so, could we just replace the double-quotes with `, making the final result be

Support in-band Emsg events targeting the player with scheme id
`urn:mpeg:dash:event:2012` and scheme values "1", "2" and "3".

Sorry for being annoying, it would look weird if only the dash piece has a different style.